build.sh

```diff
@@ -10,6 +10,34 @@ fi
 apk update
 apk add alpine-sdk util-linux strace file zlib-dev zlib-static autoconf automake libtool
 
+# Build static squashfuse
+apk add glib-dev  glib-static fuse-dev fuse-static zstd-dev zstd-static # fuse3-static fuse3-dev
+# wget -c -q "https://github.com/vasi/squashfuse/releases/download/0.1.104/squashfuse-0.1.104.tar.gz"
+wget -c -q "https://github.com/vasi/squashfuse/archive/refs/heads/master.zip"
+# tar xf squashfuse-*.tar.gz
+unzip master.zip
+cd squashfuse-*/
+./autogen.sh
+./configure --help
+./configure CFLAGS=-no-pie LDFLAGS=-static
```

Comment:
is squashfuse a library ? if so, LDFLAGS=-static isnt't needed, it will only cause binaries to be static linked, but will not result in having the build system provide a static library (.a archive)